actions/rules/action_state_to_metastate.go

```diff
@@ -447,7 +465,9 @@ func (act ActionStateToMetaState) OnStateChange(newContext convert.ChangeDetecto
 							// only fail here when the conversion fails AND the slice is non nil.
 							return nil, nil, errs.New("error converting SystemBoardcolumns set")
 						}
-						wi.Fields[workitem.SystemBoardcolumns] = append(currentSystemBoardColumn, column.ID.String())
+						if !act.contains(currentSystemBoardColumn, column.ID.String()) {
+							wi.Fields[workitem.SystemBoardcolumns] = append(currentSystemBoardColumn, column.ID.String())
```

Comment:
Is there a check somewhere that we don't append the same board number over and over?